packages/core/src/__tests__/state-manager.test.ts

```diff
@@ -36,26 +36,22 @@ const STRING_MAP_SCHEMA = {
   },
 }
 
-let ipfs: IpfsApi
-let ceramic: Ceramic
-let controllers: string[]
 
```

Comment:
the changes to this file are now so big that I've lost the ability to meaningfully review them. In the future, if you do a large refactor of a file like this, please do a separate PR with no behavior change, just the refactor, so that then there can be a follow-up PR with just the behavior change so it's easy to review the meat of what's different